01_01_Basic_Rules.txt

```diff
@@ -513,9 +513,28 @@ succeed at a check. The DC does not need to be given out by the GM, but if
 a DC seems too high, you have the right to grumble (but seriously, ask your GM if 
 something seems too difficult. There is usually a reason why tying your shoe 
 takes a DC of 70% on that particular day, etc). After you have rolled, please
-apply any and all relevant skills and Stat Bonuses to a roll, as instructed by your GM.
+apply any and all relevant skills and Stat Modifiers to a roll, as instructed by your GM.
 When rolling Investigation for example, you will likely add your PER Stat Bonus.
 
+	Critical successes occur when your roll on a skill check lands inside your

```

Comment:
nb: Should we capitalize the 's' in 'Success' and add a note in the glossary?

Reply:
I rather like just one glossary entry for Critical rather than two for fails and successes. I do see arguments both ways though.